src/main/java/Course.java

```diff
@@ -0,0 +1,22 @@
+
+public class Course {
+	private String courseName;
+	private int courseDuration;
+	
+
+	public Course(String courseName, int courseDuration) {
+		this.courseName = courseName;
+		this.courseDuration = courseDuration;
+	}
+
+
+	public String getCourseName() {
+		return courseName;
+	}
+
+
+	public int getCourseDurationHrs() {
+		return courseDuration;
+	}
```

Comment:
you can get rid of all constructors, getter, setters, to String, and any other common boilerplate code with: https://projectlombok.org/